src/gnc_attitude_controller.cpp

```diff
@@ -13,6 +13,8 @@
 #include <gnc_attitude_controller.hpp>
 #include <gnc_constants.hpp>
 
+#include <lin/generators/constants.hpp>
+
```

Comment:
Why is this necessary?

Reply:
To give us the `lin::nans<...>(...)`

Reply:
I'm going to be working on throwing some documentation together - but the code I think is much much easier to understand from the `*.hpp` files alone. I moved a lot of the BS into the `*.inl` files which people in general shouldn't have to look at.